src/main/java/orionmipt/IdGenerator.java

```diff
@@ -0,0 +1,18 @@
+package orionmipt;
```

Comment:
No need in this class, delete it.
We can use `IdentityHashMap` and `objectsId.computeIfAbsent()` method each time we need a next ID.